tests/test_southernliving.py

```diff
@@ -52,14 +51,16 @@ def test_ingredients(self):
 
     def test_instructions(self):
         return self.assertEqual(
-            [
```

Comment:
Interesting!  We already had one scraper that was returning instructions as a list? :)

Reply:
Yup! There were at least 4.